lib/Widget/evaluationForms/previsitForm/generalForm/generalForm.dart

```diff
@@ -131,7 +132,12 @@ class _GeneralFormState extends State<GeneralForm> {
                                                     widget.hn),
                                             builder: (context, snapshot) {
                                               if (!snapshot.hasData) {
-                                                return CircularProgressIndicator();
+                                                return Padding(
+                                                  padding:
+                                                      const EdgeInsets.all(8.0),
+                                                  child:
+                                                      CircularProgressIndicator(),
+                                                );
```

Comment:
May b we should make this like Dialogs

Reply:
resolved https://github.com/LaCare-KMUTT/AbdoCare-Web/pull/72/commits/19d8e9d3d91a4bbb285ec4b177e2c202ba66d6b4